### DISCUSSION TITLE
Documentation mistake for SPO "Proxy"-cmdlets

### DISCUSSION THREAD

**Discussion by n300home** [pnp/powershell#915] [General] [answered]
Hi there,

found out that Copy-PnPItemProxy and Move-PnPItemProxy was renamed to *-ItemProxy without "PnP" in the middle without any compatibility aliases. 
Could you please accordingly advise your documentation?
https://pnp.github.io/powershell/cmdlets/Copy-PnPItemProxy.html
and https://pnp.github.io/powershell/cmdlets/Move-PnPItemProxy.html

Many thanks in advance

**Comment by LeonArmston** [comment, marked as answer]
I'd argue that the Proxy cmdlets should have PnP in them been as all the other PnP cmdlets have PnP in them so you can identify which module it comes from. My guess is that that when the new PnP.PowerShell was released that this by mistake wasnt fixed to reflect PnP or at least an alias added.

![image](https://user-images.githubusercontent.com/12968962/125754197-98283f01-6b69-4fae-a031-e56930662c0e.png)

Created an Issue https://github.com/pnp/powershell/issues/916

@gautamdsheth @erwinvanhunen
